tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -1367,3 +1366,52 @@ def test_getitem_slice_stop_too_small(self, xp, sp):
     @testing.numpy_cupy_allclose(sp_name='sp')
     def test_getitem_slice_stop_too_large(self, xp, sp):
         return _make(xp, sp, self.dtype)[:, None:5]
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64, numpy.complex64, numpy.complex128],
+}))
+@testing.with_requires('scipy')
+class TestCsrMatrixSetitem(unittest.TestCase):
+
+    @testing.numpy_cupy_equal(sp_name='sp')
+    def test_setitem_int_int(self, xp, sp):
+        matrix = _make(xp, sp, self.dtype)
+        matrix[0, 0] = 10
+        assert matrix[0, 0] == 10
+
+    @testing.numpy_cupy_equal(sp_name='sp')
+    def test_setitem_complex(self, xp, sp):
+        matrix = _make_complex(xp, sp, self.dtype)
+        matrix[1, 2] = 10 + 2j
+        assert matrix[0, 0] == 10 + 2j
```

Comment:
This test won't pass

Reply:
How about this?

Reply:
Why it won't pass ?

Reply:
You are setting item [1,2] but checking [0,0]

Reply:
Sorry, haven't noticed that

Reply:
It is strange that the test passed with this in your environment

Reply:
yes, it only showed some warning but not an error. I am debugging the cause of this now.